src/schemas/friendship.ts

```diff
@@ -23,6 +23,13 @@ export enum FriendshipSceneType {
   QRCode   = 30,
 }
 
+export interface FriendshipSource{
```

Comment:
Add the following comment above the interface code:

```
/**
 * Issue #2120: https://github.com/wechaty/wechaty/pull/2120
 */
```

Reply:
got it